src/components/AgenciesPage/types/Agencies.ts

```diff
@@ -0,0 +1,19 @@
+interface AgenciesName {
+  name: string;
+}
+
+export interface Agencies extends AgenciesName {
+  type: string;
+  administrator: string;
+  country_code: string;
+  launchers: string;
+  image: string;
+  spacecraft?: string;
+  [key: string]: string | undefined;
```

Comment:
what's the purpose of this line?

Reply:
https://www.typescriptlang.org/docs/handbook/2/objects.html#index-signatures

I use it because keys were not defined as strings

Reply:
We can use one more variant for resolve this error , i had this error recently 
**My variant example:** 
interface Object extends Record<string, any>{
  name?: string;
  age?: number;
}

const object: Object = {
  name: 'Dima',
  age: 18,
};

Reply:
thank you 
but i'm not sure, that we cat use any

Reply:
We can't use any and undefined.